.gitignore

```diff
@@ -1 +1,4 @@
 .DS_Store
+
+rclcpp/\.idea/
+tags
```

Comment:
Would you mind undoing the changes to `.gitignore`? I recommend adding `.idea` to your global `core.excludesfile`.

Just curious, what creates the file or folder `tags`?

Reply:
"tags" is generated by vim-gutentags. It's a tag file.

And I'll undo the changes.